corehq/apps/locations/models.py

```diff
@@ -75,6 +75,7 @@ class LocationType(models.Model):
     code = models.SlugField(db_index=False, null=True)
     parent_type = models.ForeignKey('self', null=True)
     administrative = models.BooleanField(default=False)
+    # TODO remove.  This shouldn't be used anywhere.
     shares_cases = models.BooleanField(default=False)
```

Comment:
In case this brakes something and we need to revert.  Also this'll play a little nicer with staging - it doesn't hurt to have it here, and this way you don't need to mess with reverse migrations going between branches.